staging/src/k8s.io/apimachinery/pkg/runtime/helper.go

```diff
@@ -257,3 +257,26 @@ func (d WithoutVersionDecoder) Decode(data []byte, defaults *schema.GroupVersion
 	}
 	return obj, gvk, err
 }
+
```

Comment:
I recommend reviewing commit-by-commit - commits are self-contained and build on top of each other.